salt/journal/config/srv-journal-app-config-parameters.yml

```diff
@@ -17,3 +17,12 @@ parameters:
     crm_url: https://crm.elifesciences.org/crm/civicrm/
 
     disqus_domain: elifesciences-staging
+
+    # defaults for URL generation, should only be used in the CLI
+    {% if salt['elife.only_on_aws']() %}
+    router.request_context.host: {{ salt['elife.cfg']('project.full_hostname') }}
```

Comment:
As of this moment no, but we can introduce an explicit configuration to override it where needed.
Extracting it automatically from:
```
$ cat ~/code/builder/.cfn/contexts/journal--prod.json | jq .cloudfront
{
  "subdomains": [
    "prod--cdn-journal",
    "beta",
    "gamma"
  ],
```
is not very safe or deterministic on which value should be chosen.